addons/sys_gui/initSettings.sqf

```diff
@@ -0,0 +1,20 @@
+[
+    QGVAR(volumeColorScale),
+    "LIST",
+    [LSTRING(VolumeColorScale_DisplayName), LSTRING(VolumeColorScale_Description)],
+    "ACRE2",
```

Comment:
We have an `ACRE2 UI` component (translated) in `sys_list`.

Reply:
@mharis001 I'd move the string to `sys_core`, because `sys_list` and `sys_gui` don't depend on each other.